providers/azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/features/LoadBalancerApi.java

```diff
@@ -68,7 +69,7 @@
    @MapBinder(BindToJsonPayload.class)
    LoadBalancer createOrUpdate(@PathParam("loadbalancername") String lbName,
          @PayloadParam("location") String location, @Nullable @PayloadParam("tags") Map<String, String> tags,
-         @PayloadParam("properties") LoadBalancerProperties properties);
+         @PayloadParam("properties") LoadBalancerProperties properties, @Nullable @PayloadParam("sku") LoadBalancerSKU sku);
```

Comment:
Leave the properties parameter at the end. That is more aligned with the style of all other APIs and methods.

Reply:
Fine!